REVIEW_GUIDELINES.md

```diff
@@ -36,7 +36,7 @@ There are a few different types of pull requests that are opened against this re
 1.  A version update to an existing package in a ROS distribution.  An example of this kind of PR is [25822](https://github.com/ros/rosdistro/pull/25822).  In these types of pull requests, the package owner is updating a major, minor, patch, or debinc number in the package.  As long as the ROS distribution isn’t in a “sync freeze” and the bloom-version meets the [current minimum requirements](https://docs.ros.org/en/ros2_documentation/rolling/Guides/Releasing-a-ROS-2-package-with-bloom.html#required-tools) these PRs will be merged without further review.  If the ROS distribution is in a "sync freeze", the PR will be approved with a note saying something like “Holding for sync freeze” with a link to the relevant discourse post. Once the distribution is out of sync freeze, these PRs will then be merged.
 
 1.  A new binary package in a ROS distribution.  An example of this kind of PR is [25857](https://github.com/ros/rosdistro/pull/25857).  An entry is considered a new binary package if it has a `release` field.  These PRs should generally have been opened via bloom; if they aren’t, the reviewer will ask why the submitter hasn't used bloom.  There are two sub-categories of this kind of PR:
```

Comment:
I wonder if we can get rid of bullet point 3 completely by just rewording this introduction.  Maybe something like:

```suggestion
1.  A new binary, source, or documentation package in a ROS distribution.  An example of this kind of PR is [25857](https://github.com/ros/rosdistro/pull/25857).  In this type of PR, any combination of (`release`, `doc`, and `source`) fields are permitted.  If an entry has a `release` field, the PR should generally have been opened via bloom; if it wasn't, the reviewer will ask why.  There are two sub-categories of this kind of PR:
```